.github/workflows/release-wc-next.yml

```diff
@@ -0,0 +1,40 @@
+# This action is a copy of `release-wc-and-playground` workflow for the purpose of the` next` branch only. 
+name: Release Web-component
+
+on: 
+  release:
+    branches:
+      - next
```

Comment:
is there such a thing as a release on a given branch only?

Reply:
It doesn't work 😅 I changed to `${{ github.event.release.target_commitish }}` in refs.